src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -1523,13 +1537,27 @@ private static bool CanSimplifyNullable(INamedTypeSymbol type, NameSyntax name,
             return !name.Span.Contains(argumentDecl.Span);
         }
 
-        private static bool CanReplaceWithPredefinedTypeKeywordInContext(NameSyntax name, SemanticModel semanticModel, out TypeSyntax replacementNode, ref TextSpan issueSpan, SyntaxKind keywordKind, CancellationToken cancellationToken)
+        private static bool CanReplaceWithPredefinedTypeKeywordInContext(
+            NameSyntax name,
+            SemanticModel semanticModel,
+            out TypeSyntax replacementNode,
+            ref TextSpan issueSpan,
+            SyntaxKind keywordKind,
+            string codeStyleOptionName,
+            CancellationToken cancellationToken)
```

Comment:
formatting is fine.  It's just the mix of normal parameters *and* an out-param *and* a ref param.  just super ugly.